src/Roslyn.Suppressions/Roslyn.Suppressions/RoslynSettings.cs

```diff
@@ -42,45 +41,60 @@ public class RoslynSettings
         public string SonarProjectKey { get; set; }
         
         [JsonProperty("suppressions")]
-        public IEnumerable<SonarQubeIssue> Suppressions { get; set; }
+        public IEnumerable<SuppressedIssue> Suppressions { get; set; }
     }
 
-    ///// <summary>
-    ///// Describes a single C#/VB.NET issue that has been suppressed on the server
-    ///// </summary>
-    ///// <remarks>This class contains the subset of fields from <see cref="SonarQube.Client.Models.SonarQubeIssue"/>
-    ///// needed to match suppressed issues against "live" Roslyn issues</remarks>
-    //public class SuppressedIssue
-    //{
-    //    public SuppressedIssue(string filePath, string hash, string roslynLanguage, string roslynRuleId, IssueTextRange textRange)
-    //    {
-    //        FilePath = filePath;
-    //        Hash = hash;
-    //        RoslynLanguage = roslynLanguage;
-    //        RoslynRuleId = roslynRuleId;
-    //        TextRange = textRange;
-    //    }
+    public enum RoslynLanguage
+    {
+        Unknown = 0,
+        CSharp = 1,
+        VB = 2
+    }
```

Comment:
This enum is only used for in this assembly.
Currently, the language field is serialized as "1" or "2". I think we could tell NewtonSoft to serialize the name instead.
I'll have a quick look - if it's easy I'll do it.